frontend/javascripts/test/api/api_volume_latest.spec.js

```diff
@@ -24,24 +24,23 @@ test("setActiveCell should set the active cell id", t => {
   t.is(api.tracing.getActiveCellId(), 27);
 });
 
-test("getVolumeTool should get the current tool", t => {
-  const { api } = t.context;
-  t.is(api.tracing.getVolumeTool(), VolumeToolEnum.MOVE);
-});
```

Comment:
I think this line was removed in error. Now the next test is somehow inside this one if I see it correctly.